RelationalAI.Test/DatabaseTest.cs

```diff
@@ -155,6 +155,16 @@ public override async Task DisposeAsync()
             {
                 await Console.Error.WriteLineAsync(e.ToString());
             }
+
+            try
+            {
+                await client.DeleteDatabaseAsync($"{Dbname}-clone");
```

Comment:
This assumes magic knowledge that the test method uses this name format

Also, why are we even using a DisposeAsync call for cleaning up local resources specific to a single test method?  Just wrap the actual test code in a try/finally

Reply:
Or, cleaner, a 'using' statement

Reply:
try/finally/using refactoring is out of scope of this PR. @NRHelmi Can comment on why DisposeAsync is necessary. It might be to avoid multiple delete database calls.